Translation corrections (#408)
07th-mod/higurashi-console-arcs: Update/omot_010_4.txt

			NULL, " Or Satoko, or Rika-chan.", Line_WaitForInput);
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210100865", 256, TRUE);
	OutputLine(NULL, "…もし、結婚しようと本気で考えた相手なら、俺は店になんか来させない」",
			NULL, " ...If we were serious about getting married, I wouldn't let my partner go to a place like that.\"", Line_Normal);
	ClearMessage();

//圭一rvS14/01/210100866.「相手の気持ちを考えたら金もそうだけど、他のやつに浮気してるかも、なんて疑われるの、ごめんだからな」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#956f6e>圭一</color>", NULL, "<color=#956f6e>Keiichi</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210100866", 256, TRUE);
	OutputLine(NULL, "「相手の気持ちを考えたら金もそうだけど、他のやつに浮気してるかも、なんて疑われるの、ごめんだからな」",

> 			NULL, "\"When considering what they're thinking, part of it's the money, but I wouldn't want to suspect they're cheating on me.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Considering my partner's feelings, part of it is about the money, but I also wouldn't want to suspect they're cheating on me.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f0953d>レナ</color>", NULL, "<color=#f0953d>Rena</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 2, "ps3/s14/02/210200295", 256, TRUE);
	OutputLine(NULL, "「そのうち段々、リナさんのことが疎ましくなって、見ているだけで不愉快になってきて、家をのっとられるかも、だったらその前に！」",

> 			NULL, "\"In that time, being around Rina-san gre more and more unpleasant. Just looking at her made me feel uncomfortable. It was like she was trying to take over my house, so before that, I had to get rid of her...!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"In that time, being around Rina-san grew more and more unpleasant. Just looking at her made me feel uncomfortable. It was like she was trying to take over my house, so before that, I had to get rid of her...!\"", Line_Normal);
```
